modules/openapi-generator/src/main/resources/JavaSpring/pojo.mustache

```diff
@@ -136,9 +136,11 @@ public class {{classname}}{{#parent}} extends {{{parent}}}{{/parent}}{{^parent}}
     this.{{name}}{{#isNullable}}.get(){{/isNullable}}.add({{name}}Item);
     {{/openApiNullable}}
     {{^openApiNullable}}
+    {{#defaultValue}}
     if (this.{{name}} == null) {
       this.{{name}} = {{{defaultValue}}};
     }
+    {{/defaultValue}}
```

Comment:
what if `openApiNullable=false` & no default value?

Reply:
good catch, i just spotted the same issue with https://github.com/OpenAPITools/openapi-generator/pull/14961/.

i think we will need to create an empty container (array or set) if it's null (instead of initialising with the default value if present.

Reply:
pushed a fix. please have another look when you've time. thanks.